mailbox/event/event-rabbitmq/src/main/java/org/apache/james/mailbox/events/RabbitMQEventBus.java

```diff
@@ -68,11 +67,12 @@ public RabbitMQEventBus(RabbitMQConnectionFactory rabbitMQConnectionFactory, Eve
         this.routingKeyConverter = routingKeyConverter;
         this.retryBackoff = retryBackoff;
         this.eventDeadLetters = eventDeadLetters;
-        this.isRunning = new AtomicBoolean(false);
+        this.isRunning = false;
+        this.isRunning = false;
```

Comment:
`isStopping`?

Reply:
well, it demonstrates how badly tested the whole thing is ... thank you for catching this. Honestly, I consider this start/stop protection useless but I'm waiting for the author to come back from vacations to discuss that with him, it's the reason why I didn't invest much in tests.

Reply:
In fact, this initialization is here only for expression my intent because boolean variables are always false by default. It's probably the reason why it worked anyway.